visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.e2e.ts

```diff
@@ -31,11 +31,12 @@ describe("MapTreeViewLevel", () => {
 
 	describe("Blacklist", () => {
 		it("excluding a building should exclude it from the tree-view as well", async () => {
-			await searchPanelModeSelector.openTreeView()
-			await mapTreeViewLevel.openContextMenu()
+			await searchPanelModeSelector.toggleTreeView()
+			await mapTreeViewLevel.openFolder("/root/ParentLeaf")
+			await mapTreeViewLevel.openContextMenu("/root/ParentLeaf/smallLeaf.html")
 			await nodeContextMenu.exclude()
 
-			expect(await mapTreeViewLevel.nodeExists()).toBeFalsy()
+			expect(await mapTreeViewLevel.nodeExists("/root/ParentLeaf/smallLeaf.html")).toBeFalsy()
```

Comment:
Nit: it is nice to use a variable for strings that are reused in the same function.